FLEDGE.md

```diff
@@ -225,19 +225,36 @@ The `ads` list contains the various ads that the interest group might show.  Eac
+
+The `sizeGroups` field (optionally) contains a dictionary of named lists of ad sizes. Each ad declared above must specify a size group, saying which sizes it might be loaded at. Each named ad size is also considered a size group, so you don't need to manually define singleton size groups; for example see the `sizeGroup: 'size3'` code above.
+
+At some point in the future -  no earlier than Q1 2025 -  when the sizes are declared, the URL-size pairings will be used to prefetch k-anonymity checks to limit the configurations that can win an auction, please see [this doc](https://developer.chrome.com/docs/privacy-sandbox/protected-audience-api/feature-status/#k-anonymity). In the present implementation, only the URL is used for k-anonymity checks, not the size. When an ad with a particular size wins the auction (including in the current implementation), the size will be substituted into any macros in the URL (through `{%AD_WIDTH%}` and `{%AD_HEIGHT%}`, or `${AD_WIDTH}` and `${AD_HEIGHT}`), and once loaded into a fenced frame, the size will be used by the browser to freeze the fenced frame's inner dimensions. We therefore recommend using ad size declarations, but they are not required at this time.
+
+The `auctionServerRequestFlags` field is optional and is only used for auctions [run on an auction server](https://github.com/WICG/turtledove/blob/main/FLEDGE_browser_bidding_and_auction_API.md).
+This field contains a list of enumerated values that change what data is sent in the auction blob:
+ * The `omit-ads` enumeration causes the request to omit the `ads` and `adComponents` fields for
+this interest group from the auction blob.
+  * The `include-full-ads` enumeration causes the request to include the full ad object in place of
+anywhere in the request where a plain `adRenderId` would have been sent (such as in the `ads`
+and `adComponents` fields as well as `prevWins`). Note that `include-full-ads` is not compatible
+with the auction server, so this mode is only for debugging.
+
+All fields that accept arbitrary metadata objects (`userBiddingSignals` and `metadata` field of ads) must be JSON-serializable.
 
 All fields that specify URLs for loading scripts or JSON (`biddingLogicURL`,
 `biddingWasmHelperURL`, `trustedBiddingSignalsURL`, and `updateURL`) must: be valid HTTPS URLs, contain no credentials, and have no [fragment](https://url.spec.whatwg.org/#concept-url-fragment); be
```

Comment:
Also `decisionLogicURL` and `trustedScoringSignalsURL`, yes?

Reply:
Done.